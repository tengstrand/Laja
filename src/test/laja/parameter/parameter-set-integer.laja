
#write "target/test-output/parameter-set-integer.out"
  value = {value}
  value.class.name = {value.class.name}
#end
