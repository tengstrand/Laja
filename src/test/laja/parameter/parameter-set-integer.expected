  value = 123
  value.class.name = java.lang.Integer
